Docs/Configuration.tex

```diff
@@ -8106,6 +8106,19 @@ \subsection{Output Properties}\label{uefioutputprops}
 
 \begin{enumerate}
 
+\item
+  \texttt{InitialMode}\\
+  \textbf{Type}: \texttt{plist\ string}\\
+  \textbf{Failsafe}: \texttt{Auto}\\
+  \textbf{Description}: Selects initial \texttt{ConsoleControl} mode to use before applying \texttt{TextRenderer} option.
```

Comment:
This description is cryptic. `Start text renderer in the specified mode` may be better.